src/SamplePlugin/DeployPluginForDebug.Win-MacOs.ps1

```diff
@@ -0,0 +1,121 @@
+﻿# create a switch for the option to create or check an existing NTFS Junciton
+# Further named param must be declared here. This also leverages tap-completion on commandline.
+param (
+		[switch]$NtfsJunction = $false, # Adds the -NtfsJunction switch, default is false so the default is to copy to destDir
```

Comment:
You can also do parameter aliases like so 😊

```suggestion
		[Alias('J')]
		[switch]$NtfsJunction = $false, # Adds the -NtfsJunction switch, default is false so the default is to copy to destDir
```

Reply:
Thanks! I didn't knew that this exists.